UI/Source/AppKitExtensions/NSCollectionView+Selection.swift

```diff
@@ -3,7 +3,7 @@ import Foundation
 
 public extension NSCollectionView {
 
-    /// Sets the selection to the first item in the collection view.
+    /// Sets the selection to the first item in the collection view, and calls the delegate's selection method.
     public func selectFirstItem() {
```

Comment:
This should be fine, but one suggestion: maybe add a param to all these. e.g:
```
public func selectFirstItem(notifyingDelegate: Bool = false) {
```

That way we keep the default programmatic behavior of not invoking the delegate but letting call sites do so without the ugliness.

(I believe programmatic selection doesn't generally invoke the delegate to allow programs to understand the extra bit of information about user-reaction or not. Some people invoke work on the delegate call that can be skipped if it's invoked by the program itself. e.g. think of selecting something as a result of a navigate... can cause infinite loops without differentiating between user-invoked and program-invoked)